plugin/src/main/kotlin/com/varabyte/kobweb/intellij/project/KobwebProject.kt

```diff
@@ -84,6 +60,6 @@ class KobwebProject(
     }
 
     override fun toString(): String {
-        return "KobwebProject(name='$name', type=$type, source=${source::class.simpleName})"
+        return "KobwebProject(name=${name}, type=${type}, source=${source::class.simpleName})"
     }
 }
```

Comment:
Done.

Here I was trying to emphasize the only thing that really mattered for this class was the code container it was located in. But actually, that's probably needlessly confusing design, because if someone creates two instances of a KobwebProject with the same module but different names, they wouldn't expect them to be equal.

> [!NOTE]
> It should probably be an error to create two different Kobweb projects with the same source but different metadata about that source. However, that's probably not a problem in practice, since we control the code for creating these things.

I did I want to return only "source=Local" or "source=External" in the debug string, not "source=External(huge/path/to/some/klib/somewhere/because/its/already/clear/from/the/name)". But that's probably not worth making an exception for.